src/common/constants.ts

```diff
@@ -13,4 +13,10 @@ export const SERVICES = {
   TRACER: Symbol('Tracer'),
   METER: Symbol('Meter'),
 } satisfies Record<string, symbol>;
+
+export const JOB_TYPES = {
+  Ingestion_New: 'Ingestion_New',
```

Comment:
try get the value from config instad of hard coded

Reply:
adapt the job type configuration so it will be same as the configs in our global chart values:
![image](https://github.com/user-attachments/assets/9a789405-4adc-42fa-851c-49c9d7d4ea03)

check for existing matched factories and if it failed throw exception before the mainLoop()

Reply:
now driven from the config

Reply:
should be deleted?

Reply:
deleted